infogami/infobase/common.py

```diff
@@ -1,8 +1,8 @@
-import config
 import web
 
-from core import *
-from utils import *
+from infogami.infobase import config
+from infogami.infobase.core import *
+from infogami.infobase.utils import *
```

Comment:
Can we lose the wildcard (*) imports?  PEP8 discourages their use and they disable all _undefined name_ checking.